rapid/client/action_instance_runner.py

```diff
@@ -0,0 +1,27 @@
+from flask import Flask
+
+from rapid.client import ClientCommunicator
+from rapid.client.executor import Executor
+from rapid.lib.work_request import WorkRequest
+from rapid.client.client_configuration import ClientConfiguration
+
+
+class ActionInstanceRunner(object):
+
+    def __init__(self, rapid_config):
+        # type: (ClientConfiguration) -> None
+        self.client_config = rapid_config
+
+    def run_action_instance(self, app, action_instance_id):
+        # type: (Flask, int) -> None
+        communicator = ClientCommunicator(self.client_config.master_uri, self.client_config.quarantine_directory, app, self.client_config.verify_certs)
+        communicator.register(self.client_config)
+        request_json = communicator.get_work_request_by_action_instance_id(action_instance_id)
+        executor = Executor(WorkRequest(request_json),
```

Comment:
won't fully understand all this until I see executor (I assume it's later in the PR).